### PR TITLE
Add transparentDarkGray color option for Box, IconButton, Pog

### PR DESCRIPTION
**Pull request by cmswalker**
## Adding transparent color for Box/IconButton/Pog + docs

- [x] Documentation
- [x] Tests
- [x] (N/A) Experimental evidence (required for Masonry changes)
- [x] Accessibility checkup

Non-hover:
screen shot 2019-01-11 at 2 07 48 pm
Hover:
screen shot 2019-01-11 at 2 07 54 pm

### CODE REVIEW

**Comment by akleiner2** [comment]
Are the `yarn.lock` changes intentional?

**Comment by cmswalker** [comment]
@akleiner2 I reset the lockfile 👍

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+9.10.0
```
Comment (jennyscript) [inline .node-version]: Did you mean to add this file?

Comment (cmswalker) [reply .node-version]: i can remove it, it's for autoswapping of node versions in [node-env](https://github.com/nodenv/nodenv) nvm alternative

Comment (cmswalker) [reply .node-version]: @jennyscript removed

**Pull request merged**